src/Select.js

```diff
@@ -987,6 +989,7 @@ class Select extends React.Component {
 				valueArray,
 				valueKey: this.props.valueKey,
 				onOptionRef: this.onOptionRef,
+        uniqueSelect: this.props.type === 'unique' ? true : false,
```

Comment:
Pls check the alignment of this line and simplify to `uniqueSelect: this.props.type === 'unique'`